enterprise/frontend/src/metabase-enterprise/whitelabel/components/LandingPageWidget/index.tsx

```diff
@@ -0,0 +1 @@
+export { LandingPageWidget } from "./LandingPageWidget";
```

Comment:
this is fine, though often we just do `export * from` to handle the possible future where there are multiple exports.